### PR TITLE
Update auto-upgrade-cluster.md

### PR DESCRIPTION
**Pull request by azarboon**
Hi. Following the advice from @Nickomang in [#122436](https://github.com/MicrosoftDocs/azure-docs/pull/122436), I've created this new PR which elaborates characteristics of auto upgrade:

""You don't need to stop your workloads, create a new AKS cluster or redeploy your workloads. By enabling auto-upgrade, you can ensure your clusters are up to date seamlessly and don't miss the latest features or patches from AKS and upstream Kubernetes. ""

Kindly please sign this off.

### CODE REVIEW

**Comment by Court72** [comment]
@Nickomang

Can you review the proposed changes? 
	
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by schaffererin** [comment]
#reassign:schaffererin

File: articles/aks/auto-upgrade-cluster.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Part of the AKS cluster lifecycle involves performing periodic upgrades to the l
 
 ## Why use cluster auto-upgrade
 
-Cluster auto-upgrade provides a "set once and forget" mechanism that yields tangible time and operational cost benefits. By enabling auto-upgrade, you can ensure your clusters are up to date and don't miss the latest features or patches from AKS and upstream Kubernetes.
+Cluster auto-upgrade provides a "set once and forget" mechanism that yields tangible time and operational cost benefits. You don't need to stop your workloads, create a new AKS cluster or redeploy your workloads. By enabling auto-upgrade, you can ensure your clusters are up to date seamlessly and don't miss the latest features or patches from AKS and upstream Kubernetes.
```
Comment (schaffererin) [inline articles/aks/auto-upgrade-cluster.md:22]:
```suggestion
Cluster auto-upgrade provides a "set once and forget" mechanism that yields tangible time and operational cost benefits. You don't need to stop your workloads, redeploy your workloads, or create a new AKS cluster. By enabling auto-upgrade, you can ensure your clusters are up to date and don't miss the latest features or patches from AKS and upstream Kubernetes.
```

**Review by schaffererin — CHANGES_REQUESTED** [review]
Hi, @azarboon, thank you for your PR!

I suggested an edit that better aligns with the Microsoft writing style guidance. Once that suggestion is committed, this PR is good for sign off :)

**Comment by azarboon** [comment]
> Hi, @azarboon, thank you for your PR!
> 
> I suggested an edit that better aligns with the Microsoft writing style guidance. Once that suggestion is committed, this PR is good for sign off :)

Thank you. Committed your suggestion. Please sign it off.

**Comment by schaffererin** [comment]
#sign-off

**Comment by schaffererin** [comment]
@Court72, could you please assist me in getting this PR signed off on? :)

**Pull request merged**